### PR TITLE
chore: increase version number to 0.16.24

### PR DESCRIPTION
**Pull request by esmeetewinkel**
PR Checklist

- [ ] PR title descriptive (can be used in release notes)

## Description

PR increase version number to 0.16.24

### CODE REVIEW

File: .github/workflows/reusable-android-build.yml [hunk 1]
```diff
@@ -129,7 +129,7 @@ jobs:
             echo "Version Code: $VERSION_CODE"
             echo "Version: $VERSION"
             # This will need to change currently looking for specific version and not place holder
-            sed -i -e "s/16022/$VERSION_CODE/g" -e "s/0.16.22/$VERSION/g" ./android/app/build.gradle
```
Comment (esmeetewinkel) [inline .github/workflows/reusable-android-build.yml:132]: @jfmcquade Should this be added to the action `yarn scripts version`? This is what I normally use to bump the version but it only updates the other two places...

**Review by jfmcquade — APPROVED** [review]
I've also bumped the version number in the github workflow `reusable-android-build.yml`, which is used by content repos. Currently it's still looking for a specific version number (now increased to match the othee files) to replace with the target version number specified by the repo that is invoking the action.

File: .github/workflows/reusable-android-build.yml (hunk 1 above)
Comment (jfmcquade) [reply .github/workflows/reusable-android-build.yml:132]: Good point. Really we want to action #2136, but this is probably a good idea in the short term, I'll have a look

Comment (chrismclarke) [reply .github/workflows/reusable-android-build.yml:132]: If we're updating the android build number for content repos shouldn't we be using a variable coming from the content repo itself? E.g. the content repo version number...

**Pull request merged**